packages/session-replay-browser/README.md

```diff
@@ -62,6 +62,10 @@ Any time that the session id for the user changes, the session replay SDK must b
 ```typescript
 sessionReplay.setSessionId(UNIX_TIMESTAMP)
 ```
+You can optionally pass a new device id as a second argument as well:
+```typescript
+sessionReplay.setSessionId(UNIX_TIMESTAMP, deviceId)
```

Comment:
@kwalker3690 - Curious. Why not have a separate function to set device id. Is there a need to merge them both together?